### PR TITLE
Adding curator-operator namespace to smaug cluster

### PR DESCRIPTION
**Pull request by dystewart**
Adding namespace resources for [https://github.com/operate-first/curator-operator](https://github.com/operate-first/curator-operator) in response to [https://github.com/operate-first/support/issues/575](https://github.com/operate-first/support/issues/575)

### CODE REVIEW

**Comment by skanthed** [comment]
@dystewart Can you make sure that this gets reviewed and merged? So I can create PR for CRD's

**Review by Gregory-Pereira — COMMENTED** [review]
/lgtm

File: curator-operator/overlays/moc/smaug/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```
[the remark this answers is not in the document]
Comment (dystewart) [reply curator-operator/overlays/moc/smaug/kustomization.yaml:1]: @HumairAK @skanthed Surbhi will be making PRs to add those resources once the namespace is set up

[the remark this answers is not in the document]
Comment (HumairAK) [reply curator-operator/overlays/moc/smaug/kustomization.yaml:1]: Okay, can you remove this folder then `curator-operator`, @skanthed can create this directory in her pr if it's needed.

[the remark this answers is not in the document]
Comment (dystewart) [reply curator-operator/overlays/moc/smaug/kustomization.yaml]: @HumairAK removed

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**